benchmarks/generate_plot.py

```diff
@@ -32,17 +32,21 @@
             data=scikit_learn_results[scikit_learn_results["Method"] ==
                                       "fit"], ax=ax1, errorbar=None)
 
+y_min = 0
+y_max = 82
```

Comment:
I think we can go ahead and make this 85, in order to give `78.5` a bit more breathing room.

Reply:
Yeah that's what I shoot for originally, but tuned down a bit in case anyone prefers a more compact view 🙂

Reply:
Updated in b7e835c.

Reply:
Thanks!